app/controllers/apis_controller.rb

```diff
@@ -17,6 +17,7 @@ def create
     params[:api][:tags] = params[:api][:tags].split(',')
     api = Api.new(api_params)
     api.user = current_user
+
     if api.save
       params[:api][:tags].each do |tag|
         new_tag = Tag.new(name: "#{tag}")
```

Comment:
This controller method is getting pretty fat. Consider factoring out some of this into a method on the API object instead.